src/sidebar/components/Menu.js

```diff
@@ -186,35 +173,59 @@ export default function Menu({
       <button
         aria-expanded={isOpen ? 'true' : 'false'}
         aria-haspopup={true}
-        className="Menu__toggle"
+        className={classnames(
+          'hyp-u-outline-on-keyboard-focus',
+          'flex items-center justify-center rounded-sm transition-colors',
+          {
+            'text-grey-7 hover:text-grey-9': !isOpen,
+            'text-brand': isOpen,
```

Comment:
Does this `text-brand` actually get used anywhere? None of the menus I can see have their text change to red when open.